Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -264,8 +264,14 @@ def check_for_latest_xquartz
 
         <<-EOS.undent
           Your XQuartz (#{installed_version}) is outdated.
-          Please install XQuartz #{latest_version} (or delete it):
-            https://xquartz.macosforge.org
+          Please install XQuartz #{latest_version} (or delete the current version).
```

Comment:
Not sure about the “delete the current version” bit here. Doesn't the installer replace the current one with the new version?